src/StackTask/Program.cs

```diff
@@ -1,2 +1,78 @@
-﻿// See https://aka.ms/new-console-template for more information
-Console.WriteLine("Hello, World!");
+﻿using System;
+
+namespace StackTask
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            var stack = new STACK<char>();
+            stack.Push('r');
+            stack.Push('b');
+            stack.Push('q');
+            stack.Pop();
+            stack.Push('h');
+            // stack.Clear();
+            stack.Print();
+        }
+    }
+}
+
+public class STACK<T> 
```

Comment:
Class names should be PascalCase: `Stack`.
Also, classes should be in separate files, please move it out.

Reply:
Done